src/main/mw.c

```diff
@@ -246,24 +247,20 @@ void annexCode(void)
         rcCommand[PITCH] = rcCommand_PITCH;
     }
 
-    if (feature(FEATURE_VBAT | FEATURE_CURRENT_METER)) {
-        vbatCycleTime += cycleTime;
-        if (!(++vbatTimer % VBATFREQ)) {
-
-            if (feature(FEATURE_VBAT)) {
-                updateBatteryVoltage();
-                batteryState = calculateBatteryState();
-                //handle beepers for battery levels
-                if (batteryState == BATTERY_CRITICAL)
-                    beeper(BEEPER_BAT_CRIT_LOW);    //critically low battery
-                else if (batteryState == BATTERY_WARNING)
-                    beeper(BEEPER_BAT_LOW);         //low battery
-            }
+    if (feature(FEATURE_VBAT)) {
+        /* currentTime will rollover @ 70 minutes */
+        if ((currentTime - vbatLastServiced) >= VBATINTERVAL) {
```

Comment:
>>but it may be debugging nightmare - it takes a bit over hour to happen.

It works and has been tested as-is. The wrap around at 70 minutes is commented.

I really don't like casting unsigned/signed, it's caused me problems in the past